pkgs/by-name/re/regina-normal/update.nix

```
@@ -0,0 +1,60 @@
+{
+  lib,
+  writeShellScript,
+  coreutils,
+  findutils,
+  curl,
+  gnupg,
+  gawk,
+  jq,
+  nix,
+  common-updater-scripts,
+
+  pname ? "regina-normal",
+  version,
+  url ? "https://api.github.com/repos/regina-normal/regina/releases/latest",
+  publicKey ? ./publickey.asc, # Benjamin Burton's PGP key (https://people.debian.org/~bab/regina-key.txt or https://keybase.io/benjaminburton/pgp_keys.asc)
+}:
+
+writeShellScript "update-${pname}" ''
+  PATH="${
+    lib.makeBinPath [
+      coreutils
+      findutils
+      curl
+      gnupg
+      jq
+      gawk
+      nix
+      common-updater-scripts
+    ]
+  }"
+  set -euo pipefail
```

```suggestion

  set -euxo pipefail
```
`set -x` is useful too